cmd/info/info.go

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) 2024, Oracle and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package info
+
+import (
+	"fmt"
+	"github.com/gosuri/uitable"
+	"github.com/oracle-cne/ocne/pkg/cmdutil"
+	"github.com/spf13/cobra"
+	"os"
+)
+
+const (
+	CommandName = "info"
+	helpShort   = "Display information about OCNE "
+	helpLong    = `Display information about OCNE that may be difficult to find.`
```

Comment:
We can't imply that information about OCNE may be difficult to find. Perhaps something like, "Display helpful information about usage and configuration options available to the user", could be better.

Reply:
Thanks! I made these changes!